readthedocs/organizations/admin.py

```diff
@@ -0,0 +1,85 @@
+"""Django admin interface for organization models."""
+
+from django.contrib import admin
+from django.contrib.auth.models import User
+from django.db.models import Q
+from django.urls import reverse
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+
+from readthedocs.oauth.tasks import sync_remote_repositories
+from readthedocs.organizations.models import Organization
+
+
+class OrganizationAdmin(admin.ModelAdmin):
```

Comment:
We will need to combine this code with our "Read the Docs for Business" repository because we already have a `OrganizationAdmin` class there.